core/domain/email_manager.py

```diff
@@ -544,6 +545,59 @@ def send_role_notification_email(
         sender_name=inviter_user_settings.username)
 
 
+def send_emails_to_subscribers(creator_id, exploration_id, exploration_title):
+    """Sends an email to all the subscribers of the creators when the creator
+    publishes an exploration.
+
+    Args:
+        creator_id: str. The id of the creator who has published an exploration
+            and to whose subscribers we are sending emails.
+        exploration_id: str. The id of the exploration which the creator has
+            published.
+        exploration_title: str. The title of the exploration which the creator
+            has published.
+    """
+
+    creator_name = user_services.get_username(creator_id)
+    email_subject = ('%s has published another exploration!' % creator_name)
+    email_body_template = (
+        'Hi %s,<br>'
+        '<br>'
+        'Your subscription, %s has published a new exploration. You can play '
+        'the exploration over here - '
+        '<a href="https://www.oppia.org/explore/%s">%s</a>.<br>'
+        '<br>'
+        'Thanks, and happy learning!<br>'
+        '<br>'
+        'Best wishes,<br>'
+        '- The Oppia Team<br>'
+        '<br>%s')
+
+    if not feconf.CAN_SEND_EMAILS:
+        log_new_error('This app cannot send emails to users.')
+        return
+
+    if not feconf.CAN_SEND_SUBSCRIPTION_EMAILS:
+        log_new_error('This app cannot send subscription emails to users.')
+        return
+
+    recipient_list = subscription_services.get_all_subscribers_of_creator(
+        creator_id)
+    for recipient_id in recipient_list:
+        recipient_user_settings = user_services.get_user_settings(recipient_id)
+        recipient_preferences = (
+            user_services.get_email_preferences(recipient_id))
+
+        if recipient_preferences['can_receive_subscription_email']:
+            email_body = email_body_template % (
+                recipient_user_settings.username, creator_name, exploration_id,
+                exploration_title, EMAIL_FOOTER.value)
+            _send_email(
+                recipient_id, feconf.SYSTEM_COMMITTER_ID,
+                feconf.EMAIL_INTENT_SUGGESTION_NOTIFICATION,
```

Comment:
Why are you using suggestion intent ? You should create a new intent for subscription emails. Suggestion emails are entirely different then subscription emails.